docker/web/app/src/Foo.php

```diff
@@ -22,16 +22,16 @@ public function downloadImg(string $url, string $name, string $path = __DIR__ .
         if (file_exists($saveTo) === true){
             unlink($saveTo);
         }
-        $fp = fopen($saveTo,'w+');
-        fwrite($fp, $raw);
-        fclose($fp);
+        // $fp = fopen($saveTo,'w+');
+        // fwrite($fp, $raw);
```

Comment:
These commented out?

Reply:
yeah while I was trying to diagnose why the code wasn't working you asked me to comment those. Should I un comment?

Reply:
Yeah this was commented out while I was trying to diagnose why the code wasn't working and you suggested commenting these out.